website/content/docs/platform/k8s/helm/terraform.mdx

```diff
@@ -2,11 +2,11 @@
 layout: 'docs'
 page_title: 'Configure Vault Helm using Terraform'
 sidebar_current: 'docs-platform-k8s-terraform'
-description: |-
-  Describes how to configure the Vault Helm chart using Terraform
+description: >-
+  Describes how to configure the Vault Helm chart using Terraform.
 ---
 
-# Configuring Vault helm with terraform
+# Configuring Vault Helm with Terraform
```

Comment:
This doesn't match the page_title?

```suggestion
# Configure Vault Helm using Terraform
```

Reply:
I must have missed it somehow... Thank you for catching.